charts/camunda-platform/charts/identity/templates/deployment.yaml

```diff
@@ -58,6 +58,8 @@ spec:
             {{- end }}
           - name: KEYCLOAK_INIT_OPERATE_ROOT_URL
             value: {{ tpl .Values.global.identity.auth.operate.redirectUrl $ | quote }}
+          - name: KEYCLOAK_INIT_CONSOLE_ROOT_URL
```

Comment:
Identity fails because the url is added without the rest of the config.
in the current file (identity/templates/deployment.yaml), search for operate, and you will find all the settings needed to add a new component (you can mainly copy/paste the same as operate).

Reply:
@aabouzaid I've updated the config, and checked the failing github actions. Don't know exactly what's still wrong. Would be great if you could have a look.

Reply:
for the record, the issue was related to an unreleased change in Identity and we passed the issue by using the Identity snapshot tag.